linkerd/app/src/lib.rs

```diff
@@ -179,8 +181,11 @@ impl Config {
                     .build_server(
                         outbound_listen,
                         outbound_addr.port(),
+                        svc::stack(refine.clone())
+                            .push_map_response(|(n, _)| n)
```

Comment:
minor nit, probably not important: it seems like it _could_ be good to avoid doing all the stuff with the vec and so on in cases where we only care about the refined name? but, i don't think it's that costly.

Reply:
Yeah, agreed.. I think it's fine for now.